container_files/tutorials/Identifying biased features.ipynb

```diff
@@ -0,0 +1,1689 @@
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>previous</th>\n",
+       "      <td>-0.004830</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>day_of_week</th>\n",
+       "      <td>0.004189</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>marital</th>\n",
+       "      <td>0.003695</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>housing</th>\n",
+       "      <td>0.002859</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>loan</th>\n",
+       "      <td>0.000629</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                      explanation\n",
+       "_rowName                         \n",
+       "duration                 1.297130\n",
+       "\"\"\"emp.var.rate\"\"\"       0.526305\n",
+       "\"\"\"nr.employed\"\"\"        0.303379\n",
+       "pdays                    0.170738\n",
+       "euribor3m                0.139221\n",
+       "month                    0.135022\n",
+       "\"\"\"cons.price.idx\"\"\"     0.056744\n",
+       "poutcome                 0.043148\n",
+       "age                      0.035058\n",
+       "\"\"\"cons.conf.idx\"\"\"      0.020872\n",
+       "job                      0.015212\n",
+       "campaign                 0.013997\n",
+       "default                  0.012997\n",
+       "contact                  0.012676\n",
+       "education                0.007162\n",
+       "previous                -0.004830\n",
+       "day_of_week              0.004189\n",
+       "marital                  0.003695\n",
+       "housing                  0.002859\n",
+       "loan                     0.000629"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "mldb.query(\"\"\"\n",
+    "SELECT *\n",
+    "FROM transpose((\n",
+    "    SELECT avg({explain({features: {* EXCLUDING (y)}, label: y='yes'})[explanation] as *}) AS *\n",
+    "    NAMED 'explanation'\n",
+    "    FROM bank_raw\n",
+    "    WHERE rowHash() % 4 = 0\n",
+    "))\n",
+    "ORDER BY abs(explanation) DESC\n",
+    "\"\"\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now what is striking here is that there is one feature that really stands out: duration. This is the actual duration of the call. Clearly, that information would not be available in a real life setting: you can't know the duration of a call before it's over, and when it's over you already now if the client has subscribed or not. If you look at the [detailed description of the data][1], you can in fact see a warning saying that using that piece of information is probably a bad idea for any realistic modeling.\n",
+    "\n",
+    "Now that we have identified the cause of those suspiciously good results, let's train and test again but ignoring it.\n",
```

Comment:
but ignoring it -> , but adding `duration` to the excluded columns so that it is not used by the model.